src/univariate/continuous/burr.jl

```diff
@@ -0,0 +1,194 @@
+"""
+    Burr(k, c, λ)
+The *Burr distribution*  with shape1 `k`, shape2 `c` and scale `λ` has probability density function
+```math
+f(x; k, c, \\lambda) = \\frac{ck}{\\lambda} \\left(\\frac{x}{\\lambda}\\right)^{c-1} \\left[ 1+\\left(\\frac{x}{\\lambda}\\right)^{c} \\right]^{-k-1}, \\quad x > 0
+```
+```julia
+Burr()                  # Burr distribution with k = 1, c = 1 and λ = 1, i.e. Burr(1, 1, 1)
+Burr(k, c)              # Burr distribution with λ = 1, i.e. Burr(k, c, 1)
+Burr(k, c, λ)           # Burr distribution with shape1 k, shape2 c and scale λ
+params(d)        # Get the parameters, i.e. (k, c, λ)
+```
+External links
+* [Burr distribution on Wikipedia](https://en.wikipedia.org/wiki/Burr_distribution)
+"""
+
+
+struct Burr{T<:Real} <: ContinuousUnivariateDistribution
+    k::T # shape1
+    c::T # shape2
+    λ::T # scale
+end
+
+function Burr(k::T, c::T, λ::T; check_args=true) where {T}
+    check_args && @check_args(Burr, k > zero(k) && c > zero(c) && λ > zero(λ))
+    return Burr{T}(k, c, λ)
+end
+
+Burr(k::Real, c::Real, λ::Real) = Burr(promote(k, c, λ)...)
+Burr(k::Integer, c::Integer, λ::Integer) = Burr(float(k), float(c), float(λ))
+Burr(k::T, c::T) where {T <: Real} = Burr(k, c, one(T))
+Burr() = Burr(1.0, 1.0, 1.0, check_args=false)
```

Comment:
One could maybe even define

```suggestion
Burr() = Burr(1, 1, 1; check_args=false)
```

I also wonder if one should define a constructor with only `k`.

Reply:
Changed as commented. I also added a constructor with only `k`.